saba/main.py

```diff
@@ -362,6 +360,11 @@ def __call__(self, models, x, y, z=None, xbinsize=None, ybinsize=None, err=None,
         """
 
         tie_list = []
+
+        _models, _x, _y, _z, _xbinsize, _ybinsize, _err, _bkg = self.remove_units(models, x, y, z, xbinsize, ybinsize, err, bkg, equivalencies)
```

Comment:
Where are these _XXX quantities used? I'm sure they are, but I cannot find the line where that happens.